.github/workflows/package.yml

```
@@ -134,7 +134,9 @@ jobs:
         password: ${{ secrets.registry_password }}
 
     steps:
-      - run: apt-get install -y zstd
+      - run: |
+          apt-get update -y
```

```suggestion
          apt-get update
```
No need for "-y" for the update
but you can add -qq to avoid noise in the logs